### DISCUSSION TITLE
Is there a way to when you change to a dark age sheet the paths would change to the roads?

### DISCUSSION THREAD

**Discussion by avorial** [JohanFalt/Foundry_WoD20#889] [Q&A] [answered]
This would allow people to pick from the pull down instead of making a unique one every time.

**Comment by JohanFalt** [comment]
No that has not been implemented. I'll add this as an issue so it is not forgotten.

**Comment by JohanFalt** [comment]
Planned for v4.2.

**Comment by JohanFalt** [comment, marked as answer]
After checking into this I'll have decided to let the present function remain - you will need to set this manually and if the path doesn't exist that you want then you create a custom one.
